<launch>
  <node name="calibration_urdf_server" pkg="tool_calibration" type="calibration_urdf_server.py" output="screen">
  </node>
</launch>
